tests/StrictObjects/SuggesterTest.phpt

```diff
@@ -55,14 +57,20 @@ class SuggesterTest extends \Tester\TestCase
 	}
 
 	/**
+	 * @param string|null $expected
```

Comment:
It's probably a _feature_ so that all parameters have a correct `@param`, but it's incompatible with the _comment only non-native-types_ strategy. :/